apps/root/src/frame/components/navigation/index.tsx

```diff
@@ -357,6 +364,16 @@ const Navigation = ({ children }: React.PropsWithChildren) => {
             type: OptionsMenuOptionType.option,
           },
           ...languageOptions,
+          {
+            label: intl.formatMessage(
+              defineMessage({ description: 'useUnlimitedApproval', defaultMessage: 'Unlimited Approval' })
```

Comment:
Maybe a tooltip with some info would be useful here?

Reply:
We dont actually allow tooltips in this items, in any case, its a feature for more expert users that already know what it means, and for more common users I dont want them even touching it :P

Reply:
Btw tried with a secondary label and it just looks too big